student-web/pom.xml

```diff
@@ -1,57 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
yessir.  student-web only contained two superclasses that were moved into item-renderer.  Technically, we don't need student-common-parent any more, but I figure we can refactor that at a later date if we choose.